EventFilter/Phase2TrackerRawToDigi/plugins/ClusterToRawProducer.cc

```diff
@@ -81,6 +81,9 @@ void ClusterToRawProducer::produce(edm::Event& iEvent, const edm::EventSetup& iS
     using namespace Phase2TrackerSpecifications;
     using namespace Phase2DAQFormatSpecification;
 
+    // Retrieve collection of clusters from the event
+    edmNew::DetSetVector<Phase2TrackerCluster1D> all_clusters = iEvent.get(ClusterCollectionToken_);
```

Comment:
all_clusters should be const& if possible, to avoid wasting CPU with array copy.

Reply:
done now